src/client/views/view.html

```diff
@@ -1,4 +1,7 @@
  <div ng-controller='inputsController'>
-   Start: <input type='text' ng-model='start'/>
-   End: <input type='text' ng-model='end'/>
+   <form ng-submit='submit()'>
+     Start: <input type='text' ng-model='start'/>
+     End: <input type='text' ng-model='end'/>
```

Comment:
these input forms should be required so we don't have empty strings sent to our server.